lib/raygun.ts

```diff
@@ -570,7 +576,8 @@ class Raygun {
           host: this._host,
           port: this._port,
           useSSL: !!this._useSSL,
-          apiKey,
+          apiKey: apiKey,
+          ...(!this._batch && { timeout: this._timeout || DEFAULT_TIMEOUT }), // This can be called via batch mode, where the normal timeout is not used, so only set if batch is disabled
```

Comment:
I think that if the `RaygunOptions.timeout` and `RaygunOptions.batchTimeout` would be unified, it wouldn't be necessary to do this `!this._batch` check.